### DISCUSSION TITLE
Which model contains final results?

### DISCUSSION THREAD

**Discussion by bhavyaac** [BayraktarLab/cell2location#132] [Q&A]
Hi, thank you for your great work on this tool. Sorry if this is a naive question. I wanted to ask where I can find the results of the final trained model on the mouse brain shown in the publication (Nature Biotech 2022). On S3Bucket, I believe there are results from two separate models in the folder `tutorial/mouse_brain_visium_results`, with titles:
1.  `LocationModelLinearDependentWMultiExperiment_2experiments_59clusters_5563locations_4006genes` 
2. `LocationModelLinearDependentWMultiExperiment_59clusters_5396locations_12809genes`
Furthermore, I found more models stored in the repository [vitkl/cell2location_paper](https://github.com/vitkl/cell2location_paper).

Which of these, if any, is the final model whose results are shown in the paper, e.g. plotted in Supplementary figures 1-5?
Thank you!

**Comment by vitkl** [comment]
This folder contains results presented in the paper https://github.com/vitkl/cell2location_paper/tree/master/notebooks/selected_results/mouse_visium_snrna/scvi/v3_lr0002_Adam_signatures_c2l_Cell2location_5experiments_59clusters_14968locations_12809genes_Nonebatch_size_amortisedFalse_20000max_epochs_200alphatest

Two S3 bucket folders that you referenced are simply a result of running a default tutorial on 2 mouse brain samples (2/5).

Somehow missed this post, sorry.
